ui/src/components/AppDetailsToolbar.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { withRouter } from 'react-router-dom';
+import { Toolbar, ToolbarGroup, ToolbarItem, Breadcrumb, BreadcrumbItem, Button } from '@patternfly/react-core';
+import './AppDetailedToolbar.css';
+
+export class AppDetailsToolbar extends React.Component {
+  onDisableApp = () => {
+    console.log('Disable button clicked');
+  }
+
+  onHomeClick = () => {
+    this.props.history.push('/');
+  }
+
+  onSaveApp = () => {
+    console.log('Save button clicked');
+  }
+
+  render () {
+    return (
+      <Toolbar className='details-toolbar'>
+        <ToolbarGroup>
+          <ToolbarItem>
+            <Breadcrumb>
+              <BreadcrumbItem className='home-breadcrumb' onClick={this.onHomeClick}>Home</BreadcrumbItem>
+              <BreadcrumbItem isActive >{this.props.app.appName}</BreadcrumbItem>
+            </Breadcrumb>
+          </ToolbarItem>
+        </ToolbarGroup>
+        <ToolbarGroup className='toolbar-buttons'>
+          <Button className='toolbar-button' onClick={this.onDisableApp} variant="primary">Disable App</Button>
```

Comment:
I'm just wondering should both these buttons be the same colour, or should we distinguish them in some way from each other?

Reply:
There was another comment above about this also. Waiting on a reply from @davidffrench about whether we should change button colors. The standard used in mdc is blue so I think let's leave them like that for now so as not to block this pr.